Monika After Story/game/zz_extrasmenu.rpy

```diff
@@ -311,28 +301,43 @@ style mas_mbs_button_text is default:
 #            textbutton _("not") action NullAction()
 #            textbutton _("not3") action NullAction()
 
+style mas_extra_menu_frame:
+    background Frame("mod_assets/frames/trans_pink2pxborder100.png", Borders(2, 2, 2, 2, pad_top=2, pad_bottom=4))
+
+style mas_extra_menu_frame_dark:
+    background Frame("mod_assets/frames/trans_pink2pxborder100_d.png", Borders(2, 2, 2, 2, pad_top=2, pad_bottom=4))
+
+style mas_extra_menu_label_text is hkb_button_text:
+    color "#FFFFFF"
+
+style mas_extra_menu_label_text_dark is hkb_button_text_dark:
+    color "#FD5BA2"
+
 style mas_adjust_vbar:
     xsize 18
     base_bar Frame("gui/scrollbar/vertical_poem_bar.png", tile=False)
     thumb "gui/slider/horizontal_hover_thumb.png"
     bar_vertical True
 
-style mas_adjustable_button is default:
-    idle_background Frame("mod_assets/buttons/squares/square_idle.png", left=3, top=3)
-    hover_background Frame("mod_assets/buttons/squares/square_hover.png", left=3, top=3)
-    hover_sound gui.hover_sound
-    activate_sound gui.activate_sound
+style mas_adjust_vbar_dark:
+    xsize 18
+    base_bar Frame("gui/scrollbar/vertical_poem_bar_d.png", tile=False)
```

Comment:
```suggestion
    base_bar Frame("gui/scrollbar/vertical_poem_bar.png", tile=False)
```

Extras menu also needs to always use the stock bar, just like the scrollable menus. We can either change `mas_adjust_vbar_dark` or create a new style similar to `classroom_vscrollbar`

Reply:
Done.